java/kotlin-extractor/src/main/kotlin/KotlinFileExtractor.kt

```diff
@@ -551,10 +555,15 @@ open class KotlinFileExtractor(
         }
     }
 
-    // If `parentClassTypeArguments` is null, the parent class is a raw type.
-    private fun extractEnclosingClass(innerDeclaration: IrDeclaration, innerId: Label<out DbClassorinterface>, innerLocId: Label<DbLocation>, parentClassTypeArguments: List<IrTypeArgument>?) {
-        with("enclosing class", innerDeclaration) {
-            var parent: IrDeclarationParent? = innerDeclaration.parent
+    private fun extractEnclosingClass(
```

Comment:
It's not clear to me exactly what this function is meant to do. Could you add a doc comment for the function as a whole, please?

Reply:
Added a comment. Maybe we could change the function name to `processEnclosedClass` so that one wouldn't expect to only extract the `enclInReftype` relation. What do you think?